lib/internal/event_target.js

```diff
@@ -44,6 +45,9 @@ class Event {
 
 
   constructor(type, options) {
+    if (arguments.length === 0) {
+      throw new ERR_EVENT_NAME_REQUIRED();
```

Comment:
I think it could be useful to have a more generic error for wrong number of arguments.

Reply:
Agreed

Reply:
Sounds good to me, I'll update.